src/components/User/GroupsContainer.vue

```diff
@@ -12,7 +12,8 @@ defineProps<Props>()
 <template>
   <section :class="$style.section">
     <h2 :class="$style.title">班</h2>
-    <group-list :groups="groups" />
+    <group-list v-if="groups.length > 0" :groups="groups" />
+    <p v-else>所属している班はありません</p>
```

Comment:
確かこれって過去の分は表示せずに現在の分を表示するんでしたっけ？
```suggestion
    <p v-else>現在所属している班はありません</p>
```

Reply:
:hi_UD: